.github/workflows/npm-publish.yml

```
@@ -0,0 +1,148 @@
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref_name }}
+          name: Release ${{ github.ref_name }}
+          body: ${{ steps.get_changelog.outputs.changelog }}
+          draft: false
+          prerelease: false
+          fail_on_unmatched_files: false
+          files: |
+            ${{ steps.find_files.outputs.cjs_file }}
+            ${{ steps.find_files.outputs.esm_file }}
+            ${{ steps.find_files.outputs.types_file }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)